easybuild/easyblocks/i/imkl.py

```diff
@@ -298,6 +301,14 @@ def post_install_step(self):
                     except OSError, err:
                         raise EasyBuildError("Can't change to interface %s directory %s: %s", lib, intdir, err)
 
+                    if lib in cdftlibs and get_software_root('PGI') and lib not in patched:
+                        # Have to patch the makefile for this library
+                        run_cmd("sed -i s/gnu/pgi/g makefile", log_all=True, simple=True)
+                        run_cmd("sed -i s/gcc/pgcc/g makefile", log_all=True, simple=True)
+                        run_cmd("sed -i s/-Wall//g makefile", log_all=True, simple=True)
+                        run_cmd("sed -i s/-std=c99/-c99/g makefile", log_all=True, simple=True)
```

Comment:
these should be done with `apply_regex_substitutions`?

@ocaisa: I can try and tackle that and send you a PR, or wrap your existing PR...